.eslintrc

```diff
@@ -13,15 +13,18 @@
     }
   },
   "plugins": [
-    "prettier"
+    "prettier",
+    "react-hooks"
   ],
   "extends": [
     "eslint:recommended",
     "prettier",
     "prettier/react"
   ],
   "rules": {
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
Either make this `error` or leave it out.

Reply:
I’d copied FB’s recommended rules for this plugin, but since there’s such an extensive conversation regarding it [here](https://github.com/facebook/react/issues/14920), I’m removing it. We can always bring it back in later.